### ISSUE TITLE
Scroll ListView with Mouse will be crash app

### ISSUE THREAD

**Issue by ghost1372** [microsoft/microsoft-ui-xaml#7101]
### Describe the bug

load a listview with some data
try to scroll with mouse
you will get exception:

```
Exception thrown at 0x00007FFD86C93AAE (ntdll.dll) in AlAnvar.exe: 0xC0000005: Access violation reading location 0xFFFFFFFFFFFFFFFF.

Unhandled exception at 0x00007FFD86C9394F (ntdll.dll) in AlAnvar.exe: Indirect call guard check detected invalid control transfer.

```

### Steps to reproduce the bug

1. create a new winui 3 poject
2. load listview with data
3. scroll

### Screenshots

![Animation](https://user-images.githubusercontent.com/9213496/168477378-ec129262-6f29-4251-8b8d-a400c76179c4.gif)

### NuGet package version

WinUI 3 - Windows App SDK 1.1 Preview 3: 1.1.0-preview3

### Windows app type

- [X] UWP
- [X] Win32

### Device form factor

Desktop

### Windows version

Windows 11 (21H2): Build 22000

**Comment by DHancock** [comment]
The same exceptions are thrown when scrolling in a zoomed ScrollViewer control (from `ntdll.dll` in retail build and `Microsoft.UI.Input.dll` in debug)

**Comment by DHancock** [comment]
This is still reproducable in 1.1.0 windows app sdk (stable). The more items that are in the list, the easier it is to reproduce the exceptions.

**Comment by castorix** [comment]
This also happens with ComboBoxes (randomly, can be in a few seconds or after several minutes...)
This is annoying because ListViews or ComboBoxes are standard controls.

Test with a simple ComboBox with 200 items :

![Combobox_crash](https://user-images.githubusercontent.com/22345506/171998148-5633202a-f647-46a1-be4e-3a0108011a63.gif)

**Comment by eslyverano** [comment]
ItemsRepeater doesn't seem to have this issue [so far]. Switched to ItemsRepeater (StackLayout vertical) while this gets fixed.
Still happening in 1.1.1

**Comment by odp-remec** [comment]
The same exceptions are thrown when you have a ListView and a GridSplitter in one view and move the GridSplitter (from [WindowsCommunityToolkit](https://docs.microsoft.com/en-us/windows/communitytoolkit/controls/gridsplitter))
Happening in windows app sdk 1.1.1 (stable)
![SplitterAndListViewCrash](https://user-images.githubusercontent.com/100202704/176612748-753891ca-ad38-4b04-afef-ad07a38d9e76.gif)
As @castorix notes this is a big problem because these are standard gui elements we want to use in a productive environment.

**Comment by Balkoth** [comment]
Seeing the crash too, often times my application does not even get the chance to catch the exception, it just goes away, even when Visual Studio as debugger is attached an all exceptions are enabled.

**Comment by rfuzzo** [comment]
yep happens to me too, with just standard controls. This is app breaking.

**Comment by DHancock** [comment]
I can reproduce the `Microsoft.UI.Input.dll` exception in WindowsAppSDK 1.1.3
I've uploaded a zipped dump file to:

[https://1drv.ms/u/s!As1FsuYcnriigXNpKwdlWcm0PzdS](https://1drv.ms/u/s!As1FsuYcnriigXNpKwdlWcm0PzdS)

**Comment by gabbybilka** [comment]
Dupe of #7230 and others, the fix will be in an upcoming servicing release for 1.1.

**Comment by ghost1372** [comment]
fixed in v1.1.4
